ipn/store/kubestore/store_kube.go

```diff
@@ -20,7 +21,16 @@ import (
 )
 
 // TODO(irbekrm): should we bump this? should we have retries? See tailscale/tailscale#13024
-const timeout = 5 * time.Second
+const (
+	timeout = 30 * time.Second
```

Comment:
Bumped to 30s here just because I reuse the same context for Event emitting, but also this will be changed in a follow up where we add retries